Cleaned up PR for testing task 1

tsbonev opened the pull request:
Removed the unused classes from java_intro and merged the classes from the first test task into the master branch of the repo

mgenov commented on src/com/clouway/crm/core/Sumator/ExceptionList.java:
```diff
@@ -0,0 +1,50 @@
+import java.util.ArrayList;
+import java.util.EmptyStackException;
+
+public class ExceptionList {
```
It's an ObjectList, not an ExceptionList as it doesn't hold list of exceptions.

mgenov commented on src/com/clouway/crm/core/Sumator/ExceptionList.java:
```diff
@@ -0,0 +1,50 @@
+public class ExceptionList {
+
+    ArrayList<Object> attributes;
+    static final int SIZE = 1;
```
Doesn't look reasonable to be defined as static as it will be the same for every instance.

mgenov commented on src/com/clouway/crm/core/Sumator/ExceptionList.java:
```diff
@@ -0,0 +1,50 @@
+
+    public boolean add(Object obj){
+        try{
+            if(attributes.size() + 1 > SIZE){
```
I'm not sure that this is an expected as attributes[attributes.size()] should throw IndexOutOfBoundsException. 

e.g you can access listSize - 1 but not listSize from the list.

tsbonev commented:
Changed the out of bounds check to be == SIZE instead of one above it
Renamed the ExceptionList class to ObjectList
Removed static from the size constant

mgenov commented:
Branch has conflict and cannot be merged.

tsbonev commented:
Conflict should be resolved

mgenov commented:
<img width="803" alt="screen shot 2018-04-30 at 15 12 34" src="https://user-images.githubusercontent.com/209223/39426526-ef331b3a-4c88-11e8-9d6f-64495427390c.png">

tsbonev commented:
![image](https://user-images.githubusercontent.com/22279603/39426630-3a9f3086-4c89-11e8-9335-e33771e48e15.png)
I have no idea what the conflict might be
